nss-day-cohort-29/nutshell-midnight-dream
# Initial html element append

# Description

Set up initial formatting for chat messaging section of app

Created outline for the database

## New Feature

Please delete options that are not relevant.

- [x ] New feature (non-breaking change which adds functionality)


# Testing Instructions for Change Made

1. git fetch --all
1. git checkout initial-html-element-append
1. open browser, make sure the login page has been cleared and a chat screen appears

# Checklist:
